packages/graphql-fixtures/src/fill.ts

```diff
@@ -126,11 +126,15 @@ export type GraphQLRequest<Data, Variables, PartialData> = {
 >;
 
 const defaultResolvers = {
-  String: () => faker.random.word(),
-  Int: () => faker.datatype.number({precision: 1}),
-  Float: () => faker.datatype.number({precision: 0.01}),
+  String: () => faker.word.sample(),
```

Comment:
Oh shoot, yeah I meant to remove those tests but I might not have pushed the changes yet. After talking to the original author, those tests can be removed (they are flaky in the current version, but just luckily pass due in CI - faker v8 changed some logic for how they determine the seed and I bet that is what made them start failing)

Reply:
Done!

Reply:
The test isn't flakey. It consistently passes with the same value every time as long as you don't touch it. However is is brittle because an update to Faker's internals can result with faker having a slightly different seed value at the point of running the test after a dependency update.

It'd be fine to update the output of this test from Cat to Dog or whatever the fixture data changed to.

Reply:
I can actually make this test fail with the current version pretty easily, just by running the test suite and filtering to specific tests.

If want to make it not flaky, I think we'd want to do something like:

```
// set faker seed
const x = createFiller(...);
// set faker seed again
const y = createFiller(...);

// assert that they are the same
```